sdk/core/ci.yml

```diff
@@ -11,10 +11,7 @@ trigger:
       - sdk/core/
       - sdk/parents/azure-sdk-parent/
       - sdk/parents/azure-client-sdk-parent/
-      - eng/
-    exclude:
-      - eng/versioning
-      - eng/jacoco-test-coverage
+      - eng/pipelines/
```

Comment:
I think there might be other paths like eng/common we want to add to this list as well. 
cc @benbp @hallipr